src/controllers/ssoController.ts

```diff
@@ -58,3 +67,37 @@ export const getUserForBPD = (
       _ => ResponseSuccessJson(_)
     )
   );
+
+// TODO: We need the fn-app client to get the user profile.
+export const getUserForFIMS = (profileService: ProfileService) => (
+  req: express.Request
+): Promise<
+  | IResponseErrorValidation
+  | IResponseErrorInternal
+  | IResponseErrorTooManyRequests
+  | IResponseErrorNotFound
+  | IResponseSuccessJson<FIMSUser>
+> =>
+  withUserFromRequest(req, async user => {
+    const userProfile = await profileService.getProfile(user);
+    if (userProfile.kind !== "IResponseSuccessJson") {
+      return userProfile;
+    }
+    return withValidatedOrInternalError(
+      FIMSUser.decode({
+        acr: user.spid_level,
+        auth_time: user.created_at,
+        date_of_birth: user.date_of_birth,
+        email: fromNullable(userProfile.value.is_email_validated)
```

Comment:
Can you drop a comment line on what `email` is expected to be? Like "email is undefined if it's not validated yet"